docs/09-drawings.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Drawings
+path: /docs/drawings
+---
+
+## Sprites
+
+To create a [[Sprite]] you need to have a loaded [[Texture]] resource. You can
+then use [[Texture.asSprite]] to quickly create a [[Sprite]] or you can create
+a new instance of [[Sprite]] using the constructor. This is useful if you
+want to "slice" out a portion of an image or if you want to change the dimensions.
+
+```js
+var game = new ex.Engine();
+var txPlayer = new ex.Texture('/assets/tx/player.png');
+// load assets
+var loader = new ex.Loader([txPlayer]);
+
+// start game
+game.start(loader).then(function() {
+  // create a sprite (quick)
+  var playerSprite = txPlayer.asSprite();
+  // create a sprite (custom)
+  var playerSprite = new ex.Sprite(txPlayer, 0, 0, 80, 80);
+});
+```
+
+You can then assign an [[Actor]] a sprite through [[Actor.addDrawing]] and
+[[Actor.setDrawing]].
+
+## Sprite Sheets
+
+You can also use a [[SpriteFont]] which is special kind of [[SpriteSheet]] for use
+with [[Label|Labels]].
+
+### Creating a SpriteSheet
+
+To create a [[SpriteSheet]] you need a loaded [[Texture]] resource.
+
+```js
+const game = new ex.Engine();
+const txAnimPlayerIdle = new ex.Texture('/assets/tx/anim-player-idle.png');
+// load assets
+const loader = new ex.Loader([txAnimPlayerIdle]);
+
+// start game
+game.start(loader).then(function() {
+  const player = new ex.Actor();
+
+  // create sprite sheet with 5 columns, 1 row, 80x80 frames
+  const playerIdleSheet = new ex.SpriteSheet(txAnimPlayerIdle, 5, 1, 80, 80);
+
+  // create animation (125ms frame speed)
+  const playerIdleAnimation = playerIdleSheet.getAnimationForAll(game, 125);
+
+  // add drawing to player as "idle"
+  player.addDrawing('idle', playerIdleAnimation);
+  // add player to game
+  game.add(player);
+});
+```
+
+### Creating animations
+
+[[SpriteSheet|SpriteSheets]] provide a quick way to generate a new [[Animation]] instance.
+
+You can use _all_ the frames of a [[Texture]]([[SpriteSheet.getAnimationForAll]])
```

Comment:
The styling on the final document currently up on Netlify is weird on this element only.